src/tests/tasks.test.js

```diff
@@ -0,0 +1,89 @@
+import request from 'supertest';
+import app from '../app';
+import task from '../models/Tasks';
+import tasksSeed from '../seeds/tasks';
+
+beforeAll(async () => {
+  await task.collection.insertMany(tasksSeed);
+});
+
+let taskId;
+
+describe('POST method tests', () => {
+  test('Should not create a new Task / Error 400 / Empty body', async () => {
+    const response = await request(app).post('/tasks/').send({});
+    expect(response.status).toBe(400);
+    expect(response.body.message).toBe('Invalid input. Please check it.');
+    expect(response.body.error).toBe(true);
+  });
+
+  test('Should create a new Task', async () => {
+    const response = await request(app).post('/tasks/').send({
+      description: 'convallis nulla neque libero convallis eget eleifend',
```

Comment:
Esto lo podes declarar en una constante por fuera asi queda el codigo mas limpio. Podrias agregar antes del describe:
```suggestion
      const mockReqBody = { description: 'convallis nulla neque libero convallis eget eleifend'}
```
y en la response le mandas esa variable:
``const response = await request(app).post('/super-admins').send(mockReqBody);
```